src/pages/login/index.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import Image from 'next/image';
+import styles from "./login.module.css";
+import CustomizableAside from "../../components/customizableAside";
+
+
+const { useState } = React;
+
+function LoginForm() {
+
+    const [values, setValues] = useState({
+        email: "",
+        password: "",
+    });
+
+    const [error, setError] = useState(undefined);
+    const handleSubmit = async (e) => {
+        e.preventDefault();
+
+        try {
+            const response = await axios.post("/login", values);
+            console.log(response.data);
+            // handle success.
+        } catch (error) {
+            console.error(error);
+            // handle error (example).
+            setError("An error occurred, try again...");
+        }
+        
+
+    }
+    
+    
+    return (
+        <div className={styles.loginPage}>
+
+            <div className={styles.asideContainer}>
+            <CustomizableAside
+                    src={"/images/mainAvatar.png"}
+                    width={110}
+                    height={110}
+                    alt={"avatar"}
+                    asideCustomClass={styles.asideContain}
+                    h2Content={"Lorem ipsum lorem ipsum"}
+                    pContent={"Sed ut perspiciatis  Nemo enim ipsam voluptatem  fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt. Neque porro quisquam "}
+/>
+            </div>
+
+            <div className={styles.loginContainer}>
+                    <div className={styles.loginImage} >
+                        <Image src="/images/mainAvatar.png" width="70" height="70" priority  alt='' />
+                    </div>
+                    <form onSubmit={handleSubmit} className={styles.form}>
+                        <div className={styles.field}>
+                            <label htmlFor="email" className={styles.formLabel}>Email Address</label>
+                            <input
```

Comment:
@Sheyitofunmi I think it makes sense to add the `required` attribute to both the `email` and `password` inputs, to take advantage of built-in HTML form validations.

Reply:
alriight ma, i will do that